### PR TITLE
Windows Community Toolkit 8.0 Docs

### PR DESCRIPTION
**Pull request by michael-hawker**
Testing the initial manual import of the docs from our main repo https://github.com/CommunityToolkit/Windows

### CODE REVIEW

**Comment by michael-hawker** [comment]
image

Seen this both ways for with and without the `.md` with the `.md` is wrong, so changing back... not sure why I only see it here, maybe as it's not in `main` yet?

File: docs/windows/Collections/AdvancedCollectionView.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+* sorting your list using the `SortDirection` helper: specify any number of property names to sort on with the direction desired
+* filtering your list using a [Predicate](/dotnet/core/api/system.predicate-1): this will automatically filter your list only to the items that pass the check by the predicate provided
+* deferring notifications using the `NotificationDeferrer` helper: with a convenient _using_ pattern you can increase performance while doing large-scale modifications in your list by waiting with updates until you've completed your work
+* incremental loading: if your source collection supports the feature then AdvancedCollectionView will do as well (it simply forwards the calls)
+* live shaping: when constructing the `AdvancedCollectionView` you may specify that the collection use live shaping. This means that the collection will re-filter or re-sort if there are changes to the sort properties or filter properties that are specified using `ObserveFilterProperty`
+
+## Example
+
+```csharp
+using CommunityToolkit.WinUI;
```
Comment (michael-hawker) [inline docs/windows/Collections/AdvancedCollectionView.md:34]:
```suggestion
using CommunityToolkit.WinUI.Collections;
```

File: docs/windows/Collections/IncrementalLoadingCollection.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+`IncrementalLoadingCollection` - An extension of [ObservableCollection](/dotnet/api/system.collections.objectmodel.observablecollection-1) such that its items are loaded only when needed.
+
+## Example
+
+`IIncrementalSource` allows to define the data source:
+
+```csharp
+// Be sure to include the using at the top of the file:
+//using CommunityToolkit.WinUI;
```
Comment (michael-hawker) [inline docs/windows/Collections/IncrementalLoadingCollection.md:29]:
```suggestion
//using CommunityToolkit.WinUI.Collections;
```

**Review by michael-hawker — COMMENTED** [review]
Updating based on src change

**Review by niels9001 — APPROVED** [review]

**Pull request merged**